bigquery_etl/glean_usage/glean_app_ping_views.py

```diff
@@ -57,8 +57,8 @@ def generate_per_app(self, project_id, app_info, output_dir=None):
             (r for r in GleanPing.get_repos() if r["name"] == release_app["v1_name"])
         )
 
-        # app name is the same as the app id for the release channel: do nothing
-        if repo["app_id"] == release_app["app_name"]:
+        # app name is the same as the bq_dataset_family for the release channel: do nothing
+        if repo["bq_dataset_family"] == release_app["app_name"]:
```

Comment:
Ah, I see. The original condition would work for most things, but would fail for Firefox, where the app_id is `firefox.desktop` (and thus resolves to `firefox_desktop`).